extensions/ohif-vtk-extension/package.json

```diff
@@ -39,7 +39,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.5",
+    "css-loader": "^3.0.0",
+    "postcss-loader": "^3.0.0",
     "react-vtkjs-viewport": "0.0.9",
+    "style-loader": "^0.23.1",
```

Comment:
Do we really need three different style related loaders??

Reply:
I was taking a look and they are being used in this webpack implementation here:

https://github.com/OHIF/Viewers/blob/cb565427b3d193b85072751a92b61a2bb9171a4b/extensions/ohif-vtk-extension/webpack.config.js#L6-L20

What do you think?